.projenrc.js

```diff
@@ -0,0 +1,87 @@
+const { AwsCdkConstructLibrary, ProjectType, NodePackageManager } = require("projen");
+
+const project = new AwsCdkConstructLibrary({
+  name: "datadog-cdk-constructs",
+  description: "CDK Construct library to automatically instrument python and node functions with datadog tracing",
+  author: "Datadog",
+  authorOrganization: true,
+  entrypoint: "lib/index.js",
+  repositoryUrl: "git@github.com:DataDog/datadog-cdk-constructs.git",
+
+  projectType: ProjectType.LIB,
+  packageManager: NodePackageManager.YARN,
+
+  jsiiFqn: "projen.AwsCdkConstructLibrary",
+  defaultReleaseBranch: "main",
+  releaseEveryCommit: false,
+  cdkDependenciesAsDeps: false,
+  cdkVersion: "1.71.0",
+
+  cdkDependencies: ["@aws-cdk/aws-lambda", "@aws-cdk/aws-logs", "@aws-cdk/aws-logs-destinations", "@aws-cdk/core"],
+  devDeps: ["ts-node", "aws-cdk", "prettier", "eslint-config-prettier", "eslint-plugin-prettier"],
+  gitignore: [
+    "*.js",
+    "!jest.config.js",
+    "!src/sample",
+    "*.d.ts",
+    ".cdk.staging",
+    "cdk.out/",
+    ".parcel-cache",
+    "test/__snapshots__",
+    "default.integration.snapshot.spec.ts",
+  ],
+  npmignore: ["!LICENSE", "!LICENSE-3rdparty.csv", "!NOTICE", "/scripts"],
+  scripts: {
+    "check-formatting": "prettier --check src/**",
+  },
+  pullRequestTemplate: false,
+  dependabot: false,
+  buildWorkflow: false,
+  releaseWorkflow: false,
+  rebuildBot: false,
+  mergify: false,
+  licensed: true,
+  docgen: false,
+});
+const eslintConfig = project.tryFindObjectFile(".eslintrc.json");
+eslintConfig.addOverride("extends", [
+  "plugin:@typescript-eslint/recommended",
+  "prettier",
+  "plugin:prettier/recommended",
+]);
+
+eslintConfig.addOverride("rules", {
+  "@typescript-eslint/no-explicit-any": "off",
+  "@typescript-eslint/no-empty-interface": "off",
+  "@typescript-eslint/explicit-module-boundary-types": "off",
+  "@typescript-eslint/no-unused-vars": "off",
+  "@typescript-eslint/explicit-function-return-type": "off",
+  "no-console": "off",
+  "no-empty": "off",
+  "comma-dangle": [
+    "error",
+    {
+      imports: "ignore",
+      exports: "ignore",
+      arrays: "always-multiline",
+      functions: "always-multiline",
+      objects: "always-multiline",
+    },
+  ],
+});
+eslintConfig.addDeletionOverride("rules.quotes");
+
+const projenTasks = project.tryFindObjectFile(".projen/tasks.json");
+projenTasks.addDeletionOverride("tasks.clobber");
+projenTasks.addDeletionOverride("tasks.test:update");
+projenTasks.addDeletionOverride("tasks.bump");
+projenTasks.addDeletionOverride("tasks.release");
+projenTasks.addDeletionOverride("tasks.compat");
+const npmScripts = project.tryFindObjectFile("package.json");
+npmScripts.addDeletionOverride("scripts.clobber");
+npmScripts.addDeletionOverride("scripts.test:update");
+npmScripts.addDeletionOverride("scripts.bump");
+npmScripts.addDeletionOverride("scripts.release");
+npmScripts.addDeletionOverride("scripts.compat");
```

Comment:
The Projen API states I should be able to just write `project.removeScript("scriptName")` however that function seems to not be working correctly. I opened an issue on Projen's GitHub repo and someone is working on adding a change. https://github.com/projen/projen/issues/631. 
Once they fix it this part that removes the unnecessary scripts should be simplified to 5 calls to the .removeScript function.

Reply:
Amazing!! 🌟

Reply:
I would add a TODO comment here in the code explaining that this can be cleaned up once that issue is fixed. That way when some developer shows up in this file in the future they will understand why it's done this way and can potentially fix it if Projen is updated.